### PR TITLE
OSDOCS-11951: Test and update policies information in ROSA documentation to automatically update the policy details from JSON files

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-11951](https://issues.redhat.com//browse/OSDOCS-11951): Test and update policies information in ROSA documentation to automatically update the policy details from JSON files

Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11951

Link to docs preview:
https://83036--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-sts-about-iam-resources.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH - Can I please get QE review? Thank you!

**Review by skopacz1 — COMMENTED** [review]
Left a few comments/considerations, otherwise LGTM! Really neat use of GitHub files in our docs

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 1]
```diff
@@ -52,22 +52,7 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
...
-                    "arn:aws:iam::710019948333:role/RH-Managed-OpenShift-Installer"
-                ]
-            },
-            "Action": [
-                "sts:AssumeRole"
-            ]
-        }
-    ]
-}
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/refs/heads/master/resources/sts/4.17/sts_installer_trust_policy.json[]
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:55]: So far using variables breaks the build every time. We are trying to figure it out though.

**Review by arendej — APPROVED** [review]
lgtm

**Pull request merged**